en_ewt-ud-dev.conllu

```diff
@@ -18479,7 +18479,7 @@
 # newpar id = answers-20111106193025AA2h62V_ans-p0001
 # text = Anyone know of any HHa training in Delaware ?
 1	Anyone	anyone	PRON	NN	Number=Sing	2	nsubj	2:nsubj	_
-2	know	know	VERB	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
+2	know	know	VERB	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|Typo=Yes|VerbForm=Fin	0	root	0:root	CorrectForm=knows
```

Comment:
Actually this is not an error. This construction is short for "*Does* anyone know of...?".

Reply:
So features should perhaps be "VerbForm=Inf" ?

Reply:
Yeah